src/robusta/core/playbooks/prometheus_enrichment_utils.py

```diff
@@ -32,33 +33,47 @@ def get_node_internal_ip(node: Node) -> str:
     return internal_ip
 
 
-def create_chart_from_prometheus_query(
+def run_prometheus_query(
         prometheus_base_url: str,
         promql_query: str,
         starts_at: datetime,
-        include_x_axis: bool,
         graph_duration_minutes: int,
-        chart_title: Optional[str] = None,
-        values_format: Optional[ChartValuesFormat] = None,
-        lines: Optional[List[XAxisLine]] = []
+        end_time: datetime = None
 ):
     if not prometheus_base_url:
         prometheus_base_url = PrometheusDiscovery.find_prometheus_url()
     prom = PrometheusConnect(url=prometheus_base_url, disable_ssl=True)
-    end_time = datetime.now(tz=starts_at.tzinfo)
+    if not starts_at:
+        raise Exception("No start time specified for query.")
+    if not end_time:
+        end_time = datetime.now(tz=starts_at.tzinfo)
     alert_duration = end_time - starts_at
     graph_duration = max(alert_duration, timedelta(minutes=graph_duration_minutes))
-    start_time = end_time - graph_duration
+    start_time = end_time - max(alert_duration, graph_duration)
     resolution = 250  # 250 is used in Prometheus web client in /graph and looks good
     increment = max(graph_duration.total_seconds() / resolution, 1.0)
-    result = prom.custom_query_range(
+    logging.warning(f" resolution : {graph_duration.total_seconds()/ increment}, seconds {graph_duration.total_seconds()} inc {increment}")
```

Comment:
Why do we always print that warning?

Reply:
that was a log that needed to be removed